met-web/tests/unit/components/tenantManagement/TenantDetail.test.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { ReactNode } from 'react';
+import { render, waitFor, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import * as reactRedux from 'react-redux';
+import * as reactRouter from 'react-router';
+import * as tenantService from 'services/tenantService';
+import TenantDetail from '../../../../src/components/tenantManagement/Detail';
+import { MemoryRouter, Route, Routes, useParams } from 'react-router-dom';
```

Comment:
Just noticed this unused import :p
```suggestion
import { MemoryRouter, Route, Routes} from 'react-router-dom';
```

Reply:
Removed